src/clients/masterData/masterDataFactory.ts

```diff
@@ -0,0 +1,166 @@
+import { OutgoingHttpHeaders } from 'http'
+
+import {
+  InstanceOptions,
+  IOContext,
+  IOResponse,
+  JanusClient,
+  MasterData,
+  parseAppId,
+} from '@vtex/api'
+
+export const toStringIfNot = <K>(field: ThisType<K>) =>
+  typeof field === 'string' ? field : field?.toString()
```

Comment:
It's not necessary, try field?.toString() directly
```suggestion
export const toStringIfNot = <K>(field: ThisType<K>) =>
  field?.toString()
```

Reply:
I think it makes sense because of you Typescript, no?